src/6_branch.js

```diff
@@ -67,6 +67,10 @@ var wrap = function(parameters, func, init) {
 			 * @type {function(?Error,?): undefined}
 			 */
 			var done = function(err, data) {
+				if (err) {
```

Comment:
wait... could you do a quick manual test for me:

say we have two branch.something() functions, both don't have callback, they can be the same function.

Let's say I do this:

```
branch.somefunction(someinput1);
branch.somefunction(someinput2);
```

if I run them like this they should queue up, right? and say the first one triggers an err, the second one theoretically should still be called, because it should execute regardless of the previous items in the queue.

Would your change sustain this behavior?